public/components/marketing-cards/29.html

```diff
@@ -0,0 +1,17 @@
+<div
```

Comment:
The use of `h-` and `w-` classes being used are alarming, there are better approaches.